airflow/contrib/operators/kubernetes_operator.py

```diff
@@ -118,8 +120,8 @@ def execute(self, context):
         # if we don't a job yaml, make the job fail.
         if not self.job_yaml_string:
             raise Exception("No job yaml.")
-        else:
-            self.job_name = yaml.safe_load(self.job_yaml_string)['metadata']['name']
+
+        job_name = yaml.safe_load(self.job_yaml_string)['metadata']['name']
```

Comment:
@JLDLaughlin: Putting this on `self` seemed like an oversight. Please let me know if there was a particular reason to put this on `self` instead of a simple local variable.

Reply:
We use self.job_name to poll for completion, happy to do it another way if you have a suggestion!

Reply:
It seems self.job_name is called a few other places in the code.